cliplibs will fail if clip-glade2 doesn't find libglade2,
this workaround is very very ugly but 'acceptable' in this specific case
//mnemoc

--- ./cliplibs/Makefile.orig	2006-03-08 08:22:08.000000000 -0300
+++ ./cliplibs/Makefile	2006-03-08 08:23:03.000000000 -0300
@@ -11,7 +11,7 @@
 	do \
 		(cd $$dir ;\
 		[ -x ./configure ] && ./configure ;\
-		$(MAKE) $@ );\
+		$(MAKE) $@ || true );\
 	done
 
 clean:
